modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -505,12 +505,12 @@ def _dtypes_for_exprs(self, exprs):
     @_inherit_docstrings(PandasDataframe._maybe_update_proxies)
     def _maybe_update_proxies(self, dtypes, new_parent=None):
         if new_parent is not None:
-            super()._maybe_update_proxies(dtypes, new_parent)
+            return super()._maybe_update_proxies(dtypes, new_parent)
```

Comment:
we changed this method in `PandasDataframe` to return dtypes with an updated parent